serverless/serverless-release-notes.adoc

```diff
@@ -10,11 +10,14 @@ For an overview of {ServerlessProductName} functionality, see xref:../serverless
 
 include::modules/serverless-getting-support.adoc[leveloffset=+1]
 
+:FeatureName: Knative Eventing
```

Comment:
Is this TP block rendering where you intended?

Reply:
Yeah I think so?